gprofiler/utils/__init__.py

```diff
@@ -267,15 +282,28 @@ def run_process(
 
 
 def pgrep_exe(match: str) -> List[Process]:
-    pattern = re.compile(match)
-    procs = []
-    for process in psutil.process_iter():
-        try:
-            if pattern.match(process_exe(process)):
-                procs.append(process)
-        except psutil.NoSuchProcess:  # process might have died meanwhile
-            continue
-    return procs
+    if is_windows():
+        pythoncom.CoInitialize()
```

Comment:
Is it okay to call it multiple times? (i.e, I suppose it needs to be initialized only once)

Reply:
Yes. I have never had a problem calling this multiple times and if I remember correctly, it is needed to call this function before creating an instance of wmi.WMI